.storybook/layout.js

```diff
@@ -1,11 +1,13 @@
 import clsx from 'clsx';
 import React from 'react';
+import {useViewportSize} from '@react-aria/utils';
 
 export function VerticalCenter({children, className, style}) {
+  let size = useViewportSize();
   return (
     <div
       className={ clsx('react-spectrum-story', className) }
-      style={style}
+      style={{...style, minHeight: size.height}}
```

Comment:
100vh doesn't work as expected on iOS, so all the stories had extra scrolling. This is because the viewport changes size on scrolling when the toolbar hides but 100vh means the full screen size...